### PR TITLE
Feature/mood microservice

### PR DESCRIPTION
**Pull request by DanielBartha**
Jessica Dinova included (broken down laptop, pair programming)

### CODE REVIEW

File: backend/moods/code/controllers/moodTypesController.js
Comment (v-stamenova) [inline backend/moods/code/controllers/moodTypesController.js:1]: Naming convention in singular form - example `createMoodType`, `getAllMoodTypes`, `getMoodTypeInstanceById`

File: backend/moods/code/controllers/moodsController.js
Comment (v-stamenova) [inline backend/moods/code/controllers/moodsController.js:1]: Naming convention in singular form

File: backend/moods/code/controllers/reasonsController.js
Comment (v-stamenova) [inline backend/moods/code/controllers/reasonsController.js:1]: Naming convention in singular form

File: backend/moods/code/controllers/recordsController.js
Comment (v-stamenova) [inline backend/moods/code/controllers/recordsController.js:1]: Naming convention in singular form

**Review by v-stamenova — CHANGES_REQUESTED** [review]
Mostly the naming convention in singular form

Either unit testing (which yes please) or establish routes so that we can manually test

File: .github/workflows/backend.yml
Comment (v-stamenova) [inline .github/workflows/backend.yml:1]: I am definitely not sure about this. I need to look into it in more detail and will talk in person.

File: backend/moods/code/controllers/moodTypesController.js
Comment (JessicaDinova) [reply backend/moods/code/controllers/moodTypesController.js:1]: done

File: backend/moods/code/controllers/moodsController.js
Comment (JessicaDinova) [reply backend/moods/code/controllers/moodsController.js:1]: done

File: backend/moods/code/controllers/reasonsController.js
Comment (JessicaDinova) [reply backend/moods/code/controllers/reasonsController.js:1]: done

File: backend/moods/code/controllers/recordsController.js
Comment (JessicaDinova) [reply backend/moods/code/controllers/recordsController.js:1]: done

**Review by v-stamenova — APPROVED** [review]

**Pull request merged**